spec/unit/windows_service_spec.rb

```diff
@@ -58,6 +59,23 @@
       tempfile.unlink
     end
 
+    it "uses the default log location if none is configured" do
+      Chef::Config.merge!({:config_file => "test_config_file", :log_level => :info})
+      expect(instance).to receive(:configure_chef).twice
+      instance.service_init
+      allow(instance).to receive(:running?).and_return(true, false)
+      allow(instance.instance_variable_get(:@service_signal)).to receive(:wait)
+      allow(instance).to receive(:state).and_return(4)
```

Comment:
can we move some of these stubs up into a new before for the context or the top before block? It'd be easier to see what's unique about this test if we didn't have to compare each line.